frontend/src/app/projects/project-wrapper/project-wrapper.component.ts

```diff
@@ -49,7 +51,10 @@ export class ProjectWrapperComponent implements OnInit, OnDestroy {
         )
       )
       .subscribe({
-        next: (project: Project) => this.projectService._project.next(project),
+        next: (project: Project) => {
+          this.breadcrumbsService.updatePlaceholder({ project });
+          return this.projectService._project.next(project);
```

Comment:
```suggestion
         this.projectService._project.next(project);
```

The signature of `next` has `void` as return type. Is there a reason why you return something here?

Reply:
It's a `void` method? Okay. i just returned because the original did too.